### PR TITLE
Add a logo idea and everything that goes along

### PR DESCRIPTION
**Pull request by Cadair**
@OpenAstronomy/pyastro17-soc

### CODE REVIEW

**Comment by kelle** [comment]
Here’s a more direct link:
https://github.com/Cadair/pyastro/blob/abcd6ba02d2a28419e86fa3fc0aa344514021796/files/images/pyastro_logo_150px.png

I like it! I think we can do a bit better. I think i’d like it more if it had text associated with it. “Python in Astronomy 2017”. 
Something like this:
https://realpython.com/learn/python-first-steps/images/pythonlogo.jpg
http://www.dlr.de/sc/Portaldata/15/Resources/images/logos/PyHPC2015-Logo-2500px-transparent.png

**Comment by mwcraig** [comment]
Looks good, though having a version with the text like @kelle suggested would be nice.

**Comment by eteq** [comment]
👍  from me too on this + text.  Perhaps below, though, a la http://www.pngall.com/wp-content/uploads/2016/05/Python-Logo-PNG-Image.png ?  Then it doesn't ruin the very pretty shading that's already there.

**Comment by eteq** [comment]
One other thought, though, @Cadair: perhaps this logo should be added in the _banner_ (where the openastronomy logo is right now)?  Then the openastro logo could go at the bottom of the front page, or maybe on the other side of the banner?

Right now the presence of two logos of similar sytle is sort of confusing...

**Comment by Cadair** [comment]
I have made two different versions, one for 2017 and one more general. They are both different shapes which might be better for different situations. You can see both here: https://github.com/OpenAstronomy/pyastro/pull/3/commits/823cf3a86525592419760660d63fd8adb31ea909

I will keep the original as well, because sometimes you just want a square text-less logo (for avatars or where there is other text etc.)

**Comment by mwcraig** [comment]
👍

**Comment by dpshelio** [comment]
The logo looks very cool!!! however, I would ask PSF before creating a modification of it as it's said [in their info page](https://www.python.org/community/logos/)

**Comment by Cadair** [comment]
I am using the Python logo under the following condition:

"Use of derived logos for user groups and conferences -- Allowed if used to refer to the Python programming language. Commercial user groups and for-profit conferences require permission from the PSF."

which can be found on this page: https://www.python.org/psf/trademarks/

**Comment by dpshelio** [comment]
:+1: I really like the colour choice and the same also in the texts. Great work @Cadair

File: pages/logo.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Python in Astronomy Logo
+========================
+
+The Python in Astronomy Logo:
+
+.. image:: /images/pyastro_logo_150px.png
+   :align: center
+   :width: 150px
+
+Is based on the `Python Logo <https://www.python.org/community/logos/>`_, an SDO AIA 17.1nm EUV image courtesy of NASA/SDO and the AIA, EVE, and HMI science teams and `this <http://www.esa.int/spaceinimages/Images/2013/04/Herschel_s_view_of_the_Horsehead_Nebula>`_ image of the Horsehead nebula which is used courtesy of `ESA <http://www.esa.int/spaceinimages/ESA_Multimedia/Copyright_Notice_Images>`_.
```
Comment (astrofrog) [inline pages/logo.rst]: Is -> is

**Comment by eteq** [comment]
I really like the new logo too!  I'll go ahead and merge this so the site's up and going.

**Pull request merged**